SETUP/check_require_login.php

```diff
@@ -64,7 +64,6 @@
     "pinc/udb_user.php",
     "pinc/site_registration_protection.php",
     # 3rd party libraries
-    "pinc/functions_insert_post.php",
```

Comment:
we don't need this file anymore?When I try to access on PROD I get:
![image](https://user-images.githubusercontent.com/1526781/114253918-199d6900-9972-11eb-85c0-cde8315506d3.png)

Reply:
Nope. It was for phpBB2 support which we dropped in 2017. Removing the file is part of this MR.